draft-irtf-cfrg-vdaf.md

```diff
@@ -2167,14 +2168,36 @@ post-processing.
 We remark that, taken together, these three functionalities correspond roughly
 to the notion of "Affine-aggregatable encodings (AFEs)" from {{CGB17}}.
 
+### Multiple proofs {#multiproofs}
```

Comment:
I add this paragraph after we introduce `FLP` in details since we need these details (proof, Flp.prove, field, soundness, etc) to describe the concept of multiproofs better.  What do you think?